crypto/rsa/rsa_mp.c

```diff
@@ -106,8 +106,5 @@ int ossl_rsa_multip_cap(int bits)
     else if (bits < 8192)
         cap = 4;
 
-    if (cap > RSA_MAX_PRIME_NUM)
```

Comment:
It would be clearer if cap were initialised to `RSA_MAX_PRIME_NUM` and then reduced via the if statement.

The code, as it stands, is defensive.  A future reduction in `RSA_MAX_PRIME_NUM` won't cause a problem for example.  Hence, I'm less sure about this change.